SafeguardSessions/modules/error/ErrorCodes.pqm

```diff
@@ -0,0 +1,22 @@
+let
+    ErrorCodes.ExtendedHTTPCode = [
+        BadRequest = 4000,
+        AuthenticationError = 4010,
+        AuthorizationError = 4030,
+        NotFound = 4040,
+        SnapshotQuotaError = 4290,
+        ServerError = 5000
+    ],
+    ErrorCodes.Custom = [
+        Unexpected = 10000,

```

Comment:
#Optional

It is a good thing that Unexpected is at one end of the range of error codes. How about making it 99999?

What do you think?

Reply:
ACK.